react/week4/githubuser/src/GithubUsers.js

```diff
@@ -0,0 +1,47 @@
+import React, { useContext, useEffect } from "react";
+import { UsersContext } from "./UsersContext";
+//import GitHubRepos from "./GithubRepos";
+
+const GithubUsers = () => {
+  const userContext = useContext(UsersContext);
+
+  useEffect(() => {
+    userContext.getGitHubUserWithFetch();
+  }, [userContext.inputvalue]);
+
+  const onChangeInputValue = (e) => {
+    let value = e.target.value;
+    userContext.setInputValue(value);
+  };
+
+  return (
+    <>
```

Comment:
You don't need the fragment `<>` here, because you already have the whole component wrapped in a `<div>`.

Reply:
ya ok thank you